proseco/fid.py

```diff
@@ -36,19 +34,19 @@ def get_fid_catalog(*, detector=None, focus_offset=0, sim_offset=0,
     :param focus_offset: SIM focus offset [steps] (default=0)
     :param sim_offset: SIM translation offset from nominal [steps] (default=0)
     :param acqs: AcqTable catalog.  Optional but needed for actual fid selection.
-    :param stars: stars table.  Defaults to acqs.meta['stars'] if available.
+    :param stars: stars table.  Defaults to acqs.stars if available.
     :param dither: dither (float or 2-element sequence (dither_y, dither_z), [arcsec]
-                   Defaults to acqs.meta['dither'] if available.
+                   Defaults to acqs.dither if available.
```

Comment:
So should this have both dithers in the docstring?

Reply:
Fixed.